--- newlib-orig/config.sub	2016-12-23 03:33:54.000000000 +0100
+++ newlib/config.sub	2020-07-18 15:20:48.176686872 +0200
@@ -1399,7 +1399,7 @@
 	      | -morphos* | -superux* | -rtmk* | -rtmk-nova* | -windiss* \
 	      | -powermax* | -dnix* | -nx6 | -nx7 | -sei* | -dragonfly* \
 	      | -skyos* | -haiku* | -rdos* | -toppers* | -drops* | -es* \
-	      | -onefs* | -tirtos* | -phoenix*)
+	      | -onefs* | -tirtos* | -phoenix* | -squire*)
 	# Remember, each alternative MUST END IN *, to match a version number.
 		;;
 	-qnx*)
